src/main/java/calculator/operator/OperatorEnum.java

```diff
@@ -0,0 +1,57 @@
+package calculator.operator;
+
+public enum OperatorEnum {
+
+    PLUS("+") {
+        public double calculate(double operand1, double operand2) {
+            return operand1 + operand2;
+        }
+    },
+    MINUS("-") {
+        public double calculate(double operand1, double operand2) {
+            return operand1 - operand2;
+        }
+    },
+    MULTIPLY("*") {
+        public double calculate(double operand1, double operand2) {
+            double result = operand1 * operand2;
+            return Math.round(result * 100) / 100.0;
+        }
+    },
+    DIVISION("/") {
+        public double calculate(double operand1, double operand2) {
+            double result = operand1 / operand2;
+            return Math.round(result * 100) / 100.0;
+        }
+    };
+
+    public abstract double calculate(double operand1, double operand2);
+
+    private String keyword;
+
+    OperatorEnum(String keyword) {
+        this.keyword = keyword;
+    }
+
+    public String getKeyword() {
+        return keyword;
+    }
+
+    public static OperatorEnum getOperatorEnumFromKeyword(String operatorKeyword) {
+        for (OperatorEnum operator : OperatorEnum.values()) {
+            if (operator.getKeyword().equals(operatorKeyword)) {
+                return operator;
+            }
+        }
+        throw new IllegalArgumentException("수식 중 사칙연산 기호가 잘못되었습니다.");
```

Comment:
```suggestion
return Arrays.stream(values())
                .filter(operator -> operator.same(keyword))
                .findFirst()
                .orElseThrow(() -> new IllegalArgumentException("수식 중 사칙연산 기호가 잘못되었습니다."));
```

Stream API를 활용하여 반복문을 라이브러리 내부에 위임하고, 필터 조건/매핑 방식 등을 적극적으로 노출할 수 있습니다.